news/2020-05-15-project-loved-may-2020.md

```diff
@@ -0,0 +1,208 @@
+
+[Sound Horizon - Raijin no Hidariude](https://osu.ppy.sh/beatmapsets/45420#fruits) by [MountainBrother](https://osu.ppy.sh/users/386579)  
+172 BPM, 2:04 | 4.49★  
+The \[Kana's Demon\] difficulty is *not* being nominated for Loved.  
+*description written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)*
+
+Are you a fan of old maps? As one of the ancient osu!catch mappers, MountainBrother tried to make this map as if hyperdashes didn't exist, like the maps before the end of 2009. Although the jumps are already playable and not rare to see in regular catch maps, *Raijin no Hidariude* is still one of the most fun maps with cool hyperless jumps and asks for your precise controls. With approach rate 8, this is definitely a good map for Hidden practice. Try your best to catch all the fruits and prove yourself with this dash challenge!
+
+[![](/wiki/shared/news/2020-05-15-project-loved-may-2020/catch/2-despacito-french-version-speed-up-edit.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Sara'h - DESPACITO ( FRENCH VERSION ) (Speed Up Edit)](https://osu.ppy.sh/beatmapsets/788052#fruits) by [Vincs](https://osu.ppy.sh/users/2026299)  
+263 BPM, 2:13 | 6.33★  
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+We're back again with a map from Vincs, a French cover of the well-known song *Despacito*. The map was created as a warmup for the osu!catch World Cup 2018 for the French team. The map was used quite a bit in multiplayer lobbies back around the year and hasn't seen much sunlight lately. The song starts out slow, but it quickly builds up so that you have to move left and right all over the place, getting close to no breaks to slow down at all. In order to enjoy the full experience, make sure to turn on the hitsounds and enjoy the beauty Vincs has created.
+
+[![](/wiki/shared/news/2020-05-15-project-loved-may-2020/catch/3-fire.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[BTS - FIRE](https://osu.ppy.sh/beatmapsets/731684#fruits) by [Spectator](https://osu.ppy.sh/users/702598) and [Ollapeu](https://osu.ppy.sh/users/2064505)  
+100 BPM, 3:22 | 5.94★, 4.79★  
+*description written by [Sartan](https://osu.ppy.sh/users/4100941)*
+
+Coming in with the most fire song of this osu!catch round of Loved (hihi haha) we have another entry from our dear veteran mapper Spectator. This set features only two difficulties, but with the song being 3:22 long, there is still quite a lot to be seen. In this set we have a surprisingly nicely mapped rain by Ollapeu, somebody that has not yet been featured in Loved. There are not real weird gimmicks to be found in this set, it is very purely mapped, but that does not mean it can't pose a challenge even to the better players in our community when combined with mods such as HardRock or even DoubleTime on either of these difficulties. So go fire up your game (I did it again! Hihihi hahaha) and check it out!
```

Comment:
```suggestion
Coming in with the most fire song of this osu!catch round of Loved (hihi haha) we have another entry from our dear veteran mapper Spectator. This set features only two difficulties, but with the song being 3:22 long, there is still quite a lot to be seen. In this set we have a surprisingly nicely mapped rain by Ollapeu, somebody that has not yet been featured in Loved. 

There are not really many weird gimmicks to be found in this set as it is very purely mapped, but that does not mean it can't pose a challenge even to the better players in our community when combined with mods such as HardRock or even DoubleTime on either of these difficulties. So go fire up your game (I did it again! Hihihi hahaha) and check it out!
```